README.md

```diff
@@ -17,17 +17,83 @@ Its main features are:
 
 This code will serve as the basis for OPF on GPUs using the reduced gradient method. A similar abstraction than CuArrays will be used to port the code to AMD ROCm and Intel oneAPI through [AMDGPU.jl](https://github.com/JuliaGPU/AMDGPU.jl) and [oneAPI.jl](https://github.com/JuliaGPU/oneAPI.jl), respectively.
 
-## Installation
+## Quick-start
+### Installation
 
 ```julia
 pkg> add ExaPF
 ```
 
-## Test
+### Test
 ```julia
 pkg> test ExaPF
 ```
 
+### How to solve the power flow of a given MATPOWER instance?
+
+ExaPF implements a Newton-Raphson algorithm to solve
+the power flow equations of a power network.
+
+```julia
+# Input file
+julia> case = "case57.m"
```

Comment:
I live to copy/paste multiple lines. And my REPL doesn't automatically remove `julia>` .
```
julia> julia> case = "case57.m"
> (generic function with 1 method)
```